braindecode/models/eegsimpleconv.py

```diff
@@ -0,0 +1,148 @@
+"""
+EEG Simple Convolutional Neural Network from Yassine El Ouahidi et al. (2023).
+
+Each EEG channels is convoled individually.
```

Comment:
Maybe add a better header for the file, I added this comment temporarily.

Reply:
Done !
Thanks for the suggestions !